tests/TestCase/Controller/Component/RequestHandlerComponentTest.php

```diff
@@ -542,37 +553,44 @@ public function testStartupProcessData()
      */
     public function testStartupIgnoreFileAsXml()
     {
-        $this->Controller->request = new ServerRequest(['input' => '/dev/random']);
-        $this->Controller->request->env('REQUEST_METHOD', 'POST');
-        $this->Controller->request->env('CONTENT_TYPE', 'application/xml');
+        $this->Controller->request = new ServerRequest([
+            'input' => '/dev/random',
+            'environment' => [
+                'REQUEST_METHOD' => 'POST',
+                'CONTENT_TYPE' => 'application/xml'
+            ]
+        ]);
 
         $event = new Event('Controller.startup', $this->Controller);
         $this->RequestHandler->startup($event);
-        $this->assertEquals([], $this->Controller->request->data);
+        $this->assertEquals([], $this->Controller->request->getData());
     }
 
     /**
      * Test mapping a new type and having startup process it.
      *
+     * @group deprecated
      * @return void
      * @triggers Controller.startup $this->Controller
      */
     public function testStartupCustomTypeProcess()
     {
-        $restore = error_reporting(E_ALL & ~E_USER_DEPRECATED);
```

Comment:
please re-base 
I removed this line in other PR
`testStartupCustomTypeProcess` should not flag `@group deprecated`

Reply:
Thanks!